lib/widgets/my_point_widget.dart

```diff
@@ -0,0 +1,39 @@
+import "package:flutter/material.dart";
+
+import "../config/ui_config.dart";
+import "../theme/app_theme.dart";
+
+class MyPointWidget extends StatelessWidget {
+  const MyPointWidget({
+    required this.text,
+  });
+
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const Padding(
+          padding: EdgeInsets.only(right: DigitalGuideConfig.heightSmall),
+          child: Text(
+            "•",
+            style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
```

Comment:
If a specific textStyle isn't defined in our theme, avoid hardcoding it. Instead, use a similar one and customize it with the copyWith() method.